app/views/content/landing/social-mailing-list-sign-up.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Would probably suggest a shorter URL (eg `/landing/social-mailing-list`) , depending on what Sam suggests - more consistent with our other ones?

Reply:
Thinking about it further, perhaps we should just use `/landing/mailing-list` as it is possible that this might be used for paid search traffic in future? And the page is generic enough not to need a more specific URL?